### ISSUE TITLE
Can't use /ic menu

### ISSUE THREAD

**Issue by LowScarlet** [SkyCraft78/ItemCreator#4]
```
[06:02:15 INFO]: Low_Scarlet issued server command: /ic menu
[06:02:15 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'ic' in plugin ItemCreator v2.2.0
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.dispatchCommand(CraftServer.java:897) ~[paper-1.18.1.jar:git-Paper-177]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleCommand(ServerGamePacketListenerImpl.java:2287) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2098) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleChat(ServerGamePacketListenerImpl.java:2079) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundChatPacket.handle(ServerboundChatPacket.java:46) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundChatPacket.a(ServerboundChatPacket.java:6) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$1(PacketUtils.java:56) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.18.1.jar:git-Paper-177]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:149) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1413) ~[paper-1.18.1.jar:git-Paper-177]
        at net.minecraft.server.MinecraftServer.c(MinecraftServer.java:189) ~[paper-1.18.1.jar:git-Paper-177]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:122) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1391) ~[paper-1.18.1.jar:git-Paper-177]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1384) ~[paper-1.18.1.jar:git-Paper-177]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:132) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1362) ~[paper-1.18.1.jar:git-Paper-177]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1268) ~[paper-1.18.1.jar:git-Paper-177]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-177]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
        at info.itsthesky.itemcreator.utils.Pagination.getPage(Pagination.java:36) ~[ItemCreator.2.2.0.jar:?]
        at info.itsthesky.itemcreator.core.gui.ItemListGUI.<init>(ItemListGUI.java:75) ~[ItemCreator.2.2.0.jar:?]
        at info.itsthesky.itemcreator.core.CreatorCommand.onCommand(CreatorCommand.java:77) ~[ItemCreator.2.2.0.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        ... 21 more
```
```
[06:04:49 INFO]: This server is running Paper version git-Paper-177 (MC: 1.18.1) (Implementing API version 1.18.1-R0.1-SNAPSHOT) (Git: 2e99e5e)
You are 10 version(s) behind
Download the new version at: https://papermc.io/downloads
Previous version: git-Paper-101 (MC: 1.18.1)

[06:05:13 INFO]: ItemCreator version 2.2.0
[06:05:13 INFO]: Create custom items easily using a simple in-game GUI.
[06:05:13 INFO]: Author: ItsTheSky
```

**Comment by SkyCraft78** [comment]
Hi!
This bug has been fixed with #3, here's the jar before the actual release (will take some time because of abilities)
https://cdn.discordapp.com/attachments/941310257735532605/941316311156391936/ItemCreator.jar

**Comment by LowScarlet** [comment]
> Hi! This bug has been fixed with #3, here's the jar before the actual release (will take some time because of abilities) https://cdn.discordapp.com/attachments/941310257735532605/941316311156391936/ItemCreator.jar

still not work, same error

you need to re-open this issues
